tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,69 @@
+# Ensure that COS Lite deploys successfully, the
+# relations are setup as expected and we can access the dashboards.
+run_deploy_coslite() {
+	echo
+
+	local model_name file overlay_path admin_passwd alertmanager_ip grafana_ip prometheus_ip
+	model_name="deploy-coslite"
+	file="${TEST_DIR}/${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	overlay_path="./tests/suites/coslite/overlay"
+	juju deploy cos-lite --trust --channel=edge --overlay "${overlay_path}/offers-overlay.yaml" --overlay "${overlay_path}/storage-small-overlay.yaml"
+
+	echo "Wait for all coslite bundle charms to be active"
+	wait_for "active" '.applications["alertmanager"] | ."application-status".current' 1800
+	wait_for "active" '.applications["grafana"] | ."application-status".current' 1800
+	wait_for "active" '.applications["loki"] | ."application-status".current' 1800
+	wait_for "active" '.applications["prometheus"] | ."application-status".current' 1800
+	wait_for "active" '.applications["traefik"] | ."application-status".current' 1800
```

Comment:
Is there a reason you moved back to this way of checking the apps are active?

Reply:
I refactored to use the `active_condition` helper function.